app/survey/survey.py

```diff
@@ -140,89 +229,101 @@
 
 }
 
-WATER_SUPPLY_TEMPLATE =  [{
+WATER_SUPPLY_TEMPLATE = [{
         "question": "Which water source do you use for TYPE_WATER_USE",
         "question_id": "3",
         "possible_answers": [
-            "well_with_hand_pump",
-            "well_with_motorized_pump",
-            "protected_spring",
-            "unprotected_spring",
-            "public_tap_water",
-            "desalinated_seawater",
+            "groundwater well"
```

Comment:
the `,` at the end of the line is important to separate the components @JulianBarinton :)